src/library/scala/concurrent/SyncChannel.scala

```diff
@@ -19,12 +19,14 @@ package scala.concurrent
 @deprecated("Use `java.util.concurrent.Exchanger` instead.", since = "2.13.0")
 class SyncChannel[A] {
 
-  private[this] var pendingWrites = List[(A, SyncVar[Boolean])]()
+  private final val Signal = ()
+  private type Signal = Unit
+  private[this] var pendingWrites = List[(A, SyncVar[Signal])]()
```

Comment:
I thought I had reverted this change. It may not matter, but there's no reason to update deprecated classes. It's a candidate for reversion.